internal/cmd/group_flag.go

```diff
@@ -18,9 +18,13 @@ func addGroupFlag(cmd *cobra.Command) {
 }
 
 var fromDBFlag string
+var timestampFlag string
 
 func addFromDBFlag(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&fromDBFlag, "from-db", "", "Creates the new database based on an existing one")
 	cmd.Flags().MarkHidden("from-db")
 	cmd.RegisterFlagCompletionFunc("from-db", dbNameArg)
+
+	cmd.Flags().StringVar(&timestampFlag, "timestamp", "", "When used with --from-db option, fork will represent state of the origin database at given point in time")
```

Comment:
⛏️: please don't mention "fork" for now, since we're not using it anywhere else in the product. Some other suggestions:
```suggestion
	cmd.Flags().StringVar(&timestampFlag, "timestamp", "", "Must be used with --from-db flag. The created database will contain the data of the original one at the point in time specified by the timestamp flag")
```